presets/user/90-systemd.preset

```diff
@@ -19,3 +19,4 @@ enable systemd-tmpfiles-clean.timer
 
 disable graphical-session-pre.target
 disable graphical-session.target
```

Comment:
actually these make no sense to be listed here, they have no [Install] section. we should remove this

(unrelated to this PR, i am just noticing this)